client/tests/integration/multisignature_account.rs

```diff
@@ -12,7 +12,7 @@ use super::Configuration;
 
 #[test]
 fn transaction_signed_by_new_signatory_of_account_should_pass() -> Result<()> {
-    let (_rt, peer, iroha_client) = <PeerBuilder>::new().start_with_runtime();
+    let (_rt, peer, iroha_client) = <PeerBuilder>::new().start_with_runtime_and_port(10_605);
```

Comment:
We could have a non-sanitary declarative macro like this:
```rust
test_network! { port: 10_605, ..defaults };
```
that expands to the same and where some things **could** be customised. That way we can get rid of the methods in `test_network` that just take up space.